CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.4.0
+
+- Remove `onClose` from `SseConnection` and ensure the corresponding
```

Comment:
technically this is breaking. I'm fine ignoring since it's not likely to get noticed.

Reply:
Yup. Wasn't concerned given the low usage of this package.